lib/tasks/forms.rake

```diff
@@ -12,4 +12,13 @@ namespace :forms do
 
     puts "Updated organisation for Form: #{form.id} to be #{args[:organisation_id]}"
   end
+
+  desc "Sets the external_id for all forms to their id"
+  task set_external_ids: :environment do
+    puts "setting external_id for each form to their id"
+
+    Form.find_each { |form| form.update_columns(external_id: form.id) }
```

Comment:
I guess `update_column` here would work and might express that you are only updating a single column more clearly? But it amounts to the same thing.

Reply:
Ah good catch, didn't know there was a better variation

Reply:
Fixed